### PR TITLE
Minor fx

### PR DESCRIPTION
**Pull request by weihuoya**
make sure `PicaFSConfig` be initialized, prevent garbage data or it may generate a lot of duplicate shaders.
decrease network poll timeout, reduce network latency. use New Super Mario Bros. for test.
externals: update enet

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/5804)

### CODE REVIEW

File: src/network/room_member.cpp [hunk 1]
```diff
@@ -159,7 +159,7 @@ void RoomMember::RoomMemberImpl::MemberLoop() {
     while (IsConnected()) {
         std::lock_guard lock(network_mutex);
         ENetEvent event;
-        if (enet_host_service(client, &event, 100) > 0) {
+        if (enet_host_service(client, &event, 32) > 0) {
```
Comment (zhaowenlan1779) [inline src/network/room_member.cpp:162]: How is this number `32` chosen? Will something like 0 work better?

Comment (weihuoya) [reply src/network/room_member.cpp:162]: busy loop isn't good for performance. 1000ms / 30fps = 33.3ms, and then I choose 32.

Comment (zhaowenlan1779) [reply src/network/room_member.cpp:162]: Right, using 0 would be bad.

And I guess this is just a param to be tweaked. However currently this still means that certain packets sent may have a (maximum) 32ms delay, which I think is ... slightly too long, maybe? Definitely better than before though. I'd say we try and see how 16 or 8 works?

Comment (weihuoya) [reply src/network/room_member.cpp:162]: render frame may run in 60fps, but little games logic frame run more than 30fps.

**Comment by tywald** [comment]
Some initial testing in Monster Hunter 4 Ultimate which has been reported to have some disconnect issues as well. 

Currently I have only tested 2 players for roughly 30 minutes. My ping to the Citra room was roughly ~65ms, theirs ~228ms.

If the game was capped to 30 fps by using a cheat code, we experienced no issues. If I ran at the native 60 fps, then there would be disconnects within a few minutes. Their computer isn't fast enough to run the game at 60 fps so they kept it at 30. I'm not sure if having different fps caps for each player is the reason for the disconnect.

More testing needed. Perhaps we should try with a build where this is set to 16 instead of 32. `1000/60 = 16.6666667`.

**Comment by weihuoya** [comment]
ok, let's try 16ms

**Comment by tywald** [comment]
Was able to have a one hour session with the same person with no disconnects while I was playing at 60 fps. We did one quest when I was using 30 fps but they got disconnected after ~20min, we didn't test again when I still had 30 fps. It was probably bad luck though.

**Pull request merged**